values.yaml

```diff
@@ -147,6 +147,9 @@ auth:
     discoveryUrl: #http://<oidc_provider_address>:<oidc_provider_port>/auth/realms/<client_realm>/.well-known/openid-configuration
     clientId: #<client_name_in_oidc_provider>
     clientSecret: #<client_secret_in_oidc_provider>
+    # try to use an existing secret that has been sourced from a key vault so the clientSecret isn't stored in plaintext
+    #  if this is set then the clientSecret above is ignored.
+    existingSecret:
```

Comment:
Wh not use the same property "clientSecret" with value comes from the vault or any other tools for secrets? Personally I think it adds confusion with two properties for the same purpose.

Reply:
The way I did it is what most public Helm charts I've seen do.
It lets the dev/test people just use a plaintext password.   clientSecret is the actual password, not a reference to a k8s secret.  Also in future the existingSecret could contain more than just one key, like it may need for ldap or the simple method.

Reply:
It is also backwards compatible for existing user's config

Reply:
I've seen this pattern before too, it's pretty common